test/packages/parallel/ti_anomali/elasticsearch/transform/latest_ioc/transform.yml

```diff
@@ -22,7 +22,8 @@ description: Latest Anomali IoC data
 frequency: 30s
 sync:
   time:
-    field: "@timestamp"
+    # ensure that the field used to synchronize uses the current timestamp so it is processed
+    field: "event.ingested"
```

Comment:
Required to change this field in order to process the transforms the documents.
It looks like it requires a field whose timestamp is "now" and in order to be processed (as if they were ingested at that moment).

In the case of this package and this data stream, the value of the `@timestamp` field is a date of five years ago and it does not process the documents.

Reply:
> whose timestamp is "now"

Being "now" the last 120s, right?

Reply:
We will have to check if this happens in actual packages. I was thinking that the retention policy was also based on the `@timestamp`. Not sure if there can be cases where this change produces some kind of explosion of new docs :thinking: